astroquery/gemini/__init__.py

```diff
@@ -18,6 +18,6 @@ class Conf(_config.ConfigNamespace):
 
 conf = Conf()
 
-from .core import ObservationsClass, Observations
+from .core import GeminiObservationsClass, GeminiObservations, Observations, ObservationsClass
 
-__all__ = ['Observations', 'ObservationsClass', 'conf']
+__all__ = ['Observations', 'ObservationsClass', 'GeminiObservations', 'GeminiObservationsClass', 'conf']
```

Comment:
I think we don't recommend wildchar imports, so removing the old class names from the namespace is probably safe.